inc/cleantalk-public-integrations.php

```diff
@@ -1085,6 +1085,9 @@ function ct_preprocess_comment($comment)
 
         if ( isset($_POST['url']) && ! empty($_POST['url']) && $post_info['comment_type'] === 'comment' && isset($_POST['comment_post_ID']) ) {
             $honeypot_field = 0;
+            // if url is filled then pass them to $base_call_data as additional fields
+            $base_call_data['sender_info']['honeypot_field_value']  = $_POST['url'];
+            $base_call_data['sender_info']['honeypot_field_source'] = 'url';
```

Comment:
We have a Post class to getting any _POST variables. Use it instead of direct _POST access.